backend/lcfs/web/api/user/schema.py

```diff
@@ -28,7 +30,6 @@ class Config:
 
 
 class UserCreate(UserBase):
-    keycloak_user_id: str
-    keycloak_email: str
-    keycloak_username: str
-    pass
+    keycloak_user_id: Optional[str] = None
```

Comment:
we shouldn't have the keycloak_user_id here. It should only be used for user account mapping in the backend.